Solutions/Marain.Workflow.Abstractions/Marain/Workflows/Workflow.cs

```diff
@@ -70,41 +70,39 @@ public Workflow(string id = null, string displayName = null, string description
         public string PartitionKey => this.Id;
 
         /// <summary>
-        ///     Gets or sets the ETag of the workflow.
+        /// Gets or sets the ETag of the workflow.
         /// </summary>
         public string ETag { get; set; }
 
         /// <summary>
-        ///     Gets or sets the id of this workflow's initial State.
+        /// Gets or sets the id of this workflow's initial State.
         /// </summary>
         /// <remarks>
-        ///     Do not set directly; prefer calling <see cref="SetInitialState(WorkflowState)" />.
-        ///     When new instances of the workflow are created, they will enter this state and any entry
-        ///     conditions and actions on the state will be validated/executed. Failure in any condition
-        ///     or action will result in the new workflow instance being Faulted.
+        /// Do not set directly; prefer calling <see cref="SetInitialState(WorkflowState)" />.
+        /// When new instances of the workflow are created, they will enter this state and any entry
+        /// conditions and actions on the state will be validated/executed. Failure in any condition
+        /// or action will result in the new workflow instance being Faulted.
         /// </remarks>
         public string InitialStateId { get; set; }
```

Comment:
Should we not just make this setter private and ensure that serialization supports private setters?

Reply:
Potentially, yes. Although we're using the standard Corvus serialization settings so we should consider if this is something we should be using across the board. Once we've got this merged back I'll create an issue to further discuss this.

Reply:
I believe we should - there are loads of places I want read only properties and they are only read/write because serialization.